changelog.txt

```diff
@@ -53,6 +53,7 @@ Removed
  - The ``--workspace`` option for the command line ``job`` subcommand (#752).
  - The ability to call ``Project.workspace``, it is strictly a property now (#752).
  - ``ProjectSchema.__call__``, ``ProjectSchema.detect`` (#752).
+ - Public support for the ``synced_collection`` package (#807, #790).
```

Comment:
```suggestion
 - The public API of the ``synced_collection`` subpackage is no longer accessible (#807, #790). The `synced_collections` code will be separated into its own package in the future.
```

Reply:
I'm going to accept the first half of this suggestion but not the second. I don't want to codify promises about a new package in our changelog.